src/Components/CategoryList/CategoryList.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Category from '../Category/Category.js';
+import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+
+import './CategoryList.scss';
+
+function CategoryList({ categories }) {
+  return (
+    <Container className="category-list" fluid>
+      <Row>
+          {categories.map((element, index) =>
+            <Category key={`category-${ index }`} {...element} />
```

Comment:
Tutaj można owrapować to dodatkowo w `Col` bootstrapowy i wtedy miałbyś w tym miejscu kontrolę nad sposobem wyświetlania tych kategorii (ile kategorii w danym wierszu). Bo teraz sterujesz tym poniekąd w komponencie Kategorii, a to trochę zaburza atomowość tego komponentu.